src/bindings/python/src/pyopenvino/utils/utils.cpp

```diff
@@ -417,6 +421,8 @@ ov::Any py_object_to_any(const py::object& py_obj) {
         return py_obj.cast<double>();
     } else if (py::isinstance(py_obj, float_32_type)) {
         return py_obj.cast<float>();
+    } else if (py::isinstance(py_obj, py::module_::import("enum").attr("Enum"))) {
+        return std::make_shared<py::object>(std::move(py_obj));
```

Comment:
`py_obj` is const ref. I don't think `std::move()` does anything here.
I guess this is also the part corresponding to
> Fix: Transfer the ownership of python `enum.Enum` object

Ownership transfer suggests that the object is destroyed after `openvino.Model` is destroyed even if there are other python references to the object. That doesn't look like intended behavior.

Reply:
I removed `std::move`. You're right, actual python object should not be destroyed after `openvino.Model`.